app/views/BtaPartial.scala.html

```diff
@@ -34,6 +35,6 @@
 
     @p(id=Some("it-quarterly-reporting-p2"),content=Html(messages("bta_partial.p2")))
 
-    <a id="it-quarterly-reporting-home-button" class="button" href="@{appConfig.itvcFrontendEnvironment + controllers.routes.HomeController.home.url}" role="button">@messages("bta_partial.home_button")</a>
+    @button(id=Some("it-quarterly-reporting-home-button"), href={appConfig.itvcFrontendEnvironment + controllers.routes.HomeController.home.url}, text=messages("bta_partial.home_button"))
```

Comment:
Note: the TestBTAPartial is not migrated which makes harder to assess plausibility of changes in BtaPartial.